src/compiler/crystal/command/playground.cr

```diff
@@ -34,6 +34,10 @@ class Crystal::Command
       end
     end
 
+    playground_url = "http://#{server.host || "127.0.0.1"}:#{server.port || "8080"}"
```

Comment:
What about simply using `localhost`?
```suggestion
    playground_url = "http://#{server.host || "localhost"}:#{server.port || "8080"}"
```
That would be better especially for people who are on Linux and don't have `xdg-open` installed. (The people who have to manually open the tab). They can easily remember `localhost:8080` instead of remembering that number.

Reply:
This was done in d0c8a6bc34a8b1aad3473dd48928fc871bcfc59d. 

Didn't realize I could just commit your suggestion. New feature of Github I guess. :)